cmd/zpool/zpool_main.c

```diff
@@ -207,7 +207,7 @@ static const char *
 get_usage(zpool_help_t idx) {
 	switch (idx) {
 	case HELP_ADD:
-		return (gettext("\tadd [-fn] [-o property=value] "
+		return (gettext("\tadd [-fgLnp] [-o property=value] "
```

Comment:
There was a best effort made to keep the options all in alphabetical order.  That's not always true to to previous mistakes, but that's the intent.